Example/DApp/Modules/Sign/SessionAccount/SessionAccountPresenter.swift

```diff
@@ -41,14 +44,21 @@ final class SessionAccountPresenter: ObservableObject {
         do {
             let requestParams = try getRequest(for: method)
             
-            let request = Request(topic: session.topic, method: method, params: requestParams, chainId: Blockchain(sessionAccount.chain)!)
+            let ttl: TimeInterval = 300
+            let request = try Request(topic: session.topic, method: method, params: requestParams, chainId: Blockchain(sessionAccount.chain)!, ttl: ttl)
             Task {
                 do {
+                    await ActivityIndicatorManager.shared.start()
```

Comment:
async for start() is make sense if we're using @MainActor inside ActivityIndicatorManager. But I do not see it